### PR TITLE
[planet-finance] Strategy Update

### PR DESCRIPTION
**Pull request by ash-greninja-poke**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Comment by ash-greninja-poke** [comment]
@bonustrack @ChaituVR please review the new strategy.

**Comment by ash-greninja-poke** [comment]
@ChaituVR @bonustrack will you please start the reviewing process

**Comment by ash-greninja-poke** [comment]
@ChaituVR @bonustrack guys please merge it seems like test cases has passes successfully.

File: src/strategies/planet-finance/index.ts [hunk 1]
```diff
@@ -29,40 +29,105 @@ const planetFinanceFarmAbi = [
...
+    {
+      constant: true,
+      inputs: [{ internalType: 'address', name: '', type: 'address' }],
+      name: 'balanceOf',
+      outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+      payable: false,
+      stateMutability: 'view',
+      type: 'function',
+    },
+  ]
```
Comment (ChaituVR) [inline src/strategies/planet-finance/index.ts:54]:
```suggestion
export const bep20Abi: any = [
  'function totalSupply() view returns (uint256)',
  'function balanceOf(address) view returns (uint256)'
]
```

File: src/strategies/planet-finance/index.ts [hunk 2]
```diff
@@ -29,40 +29,105 @@ const planetFinanceFarmAbi = [
...
+			{
+				"internalType": "uint256",
+				"name": "lastUserActionTime",
+				"type": "uint256"
+			}
+		],
+		"stateMutability": "view",
+		"type": "function"
+	},
+]
```
Comment (ChaituVR) [inline src/strategies/planet-finance/index.ts:117]: Please change this to 
```
[  
  'function balanceOf() view returns (uint256)',
  'function totalShares() view returns (uint256)',
  'function userInfo(address) view returns (uint256 shares, uint256 lastDepositedTime, uint256 cakeAtLastUserAction, uint256 lastUserActionTime)' 
]
  ```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ash-greninja-poke** [comment]
@ChaituVR request changes are now done

**Comment by ChaituVR** [comment]
LGTM

**Pull request merged**